### DISCUSSION TITLE
Can you predict block hash with future eos block numbers?

### DISCUSSION THREAD

**Discussion by cool119** [community/community#66470] [Programming Help]
### Body

Many researchers say that eos blockchain is similar to blackjack's card counting

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hafiz-muhammad** [comment]
No, it is not possible to predict the block hash of the next block in EOS. The block hash is generated using a cryptographic hash function and is random and impossible to predict.

**Reply by cool119** [reply]
THX:)

**Reply by hafiz-muhammad** [reply]
You're welcome.
